corehq/blobs/s3db.py

```diff
@@ -41,10 +42,16 @@ def __init__(self, config):
         self.db.meta.client.meta.events.unregister('before-sign.s3', fix_s3_host)
 
     def report_timing(self, action):
+        def record_long_request(duration):
+            if duration > 30:
+                notify_exception(None, "S3BlobDB request took a long time.", details={
+                    'duration': duration,
```

Comment:
What do you think of adding more details here like the blob id, blob bucket, action, and possibly blob size (if known)? I'm curious how this alert will be useful or actionable if it doesn't include identifying information about the blob.

Reply:
Yeah that's definitely a good call. I think I'll also bump the threshold up to something even more absurd like 100s so we get fewer notifications. We have already seen a handful of requests come in at over 100s so those would be a good place to start getting more granular info on.

Reply:
The specific blob that took a long time cannot be identified yet, and I think that info would be very helpful in investigating why it took a long time. The blob identifier and bucket (blobdb bucket, not s3 bucket) are available everywhere `report_timing` is called. What do you think of changing the signature to `def report_timing(action, identifier, bucket):`? That would allow you to add those things to the details here.